Makefile

```diff
@@ -18,6 +18,24 @@ binary: dist FORCE
 	$(GO) version
 	$(GO) build -trimpath -ldflags "$(LDFLAGS)" -o dist/bee ./cmd/bee
 
+.PHONY: binary-file
```

Comment:
i would call this `bee-file` so that the end result is `make bee-file`

Reply:
merge into binaries